src/workerd/jsg/jsvalue.h

```diff
@@ -251,6 +251,7 @@ class JsString final: public JsBase<v8::String, JsString> {
   };
 
   template <typename T>
+    requires(std::same_as<T, uint16_t> || std::same_as<T, kj::byte>)
```

Comment:
To improve readability, this could be turned into a concept.

```cpp
template <typename T>
concept codepoint =
std::is_same_v<T, uint16_t> || std::is_same_v<T, kj::byte>;
```

(Defined once.)

And then it could be used as...

```cpp
template <codepoint T>
void fun(T& x) {}
```

In this instance, it won't do much to help the code, but long term, it could be helpful.

Reply:
Nice! Thanks for the tip.